### PR TITLE
Working on Nested mail merge docs

### PR DESCRIPTION
**Pull request by DKaramfilov**

### CODE REVIEW

File: libraries/radwordsprocessing/editing/mail-merge.md [hunk 1]
```diff
@@ -93,6 +93,109 @@ __Example 4__ performs the mail merge operation over a previously defined templa
 	RadFlowDocument mailMergeResult = document.MailMerge(mailMergeDataSource);
 {{endregion}}
 
+## Nested Mail Merge
+
+The nested mail merge functionality is supported from R1 2022. It allows you to merge data sources that contain nested data. For example your business object can contain a list of other objects and this functionality allows accessing the properties of the underlying objects. In order to use the underling objects you need to  declare a group. Currently the following group tags are supported.
```
Comment (vladislav-todorov) [inline libraries/radwordsprocessing/editing/mail-merge.md:98]:
```suggestion
The nested mail merge functionality is supported from R1 2022. It allows you to merge data sources that contain nested data. For example, your business object can contain a list of other objects and this functionality allows accessing the properties of the underlying objects. In order to use the underlying objects, you need to  declare a group. Currently, the following group tags are supported. 
```

File: libraries/radwordsprocessing/editing/mail-merge.md [hunk 2]
```diff
@@ -93,6 +93,109 @@ __Example 4__ performs the mail merge operation over a previously defined templa
...
+    }
+    public class Player
+    {
+        public string FirsName { get; set; }
+        public string LastName { get; set; }
+    }
+
+{{endregion}}
+
+Now you need to add the fields using the specific supported names. In this example we are adding the fields to the table an we will use the TableStart/TableEnd tags, but this is not mandatory and you can use tag of your choosing.
```
Comment (vladislav-todorov) [inline libraries/radwordsprocessing/editing/mail-merge.md:150]:
```suggestion
Now you need to add the fields using the specific supported names. In this example we are adding the fields to the table and we will use the TableStart/TableEnd tags, but this is not mandatory and you can use a tag of your choosing.
```

File: libraries/radwordsprocessing/editing/mail-merge.md [hunk 3]
```diff
@@ -93,6 +93,109 @@ __Example 4__ performs the mail merge operation over a previously defined templa
...
+    editor.InsertField("MERGEFIELD LastName", "");
+    editor.InsertField("MERGEFIELD TableEnd:Players", "");
+    
+    RadFlowDocument mailMergeResult = document.MailMerge(GetTeams());
+
+{{endregion}}
+
+### One Row vs Multiline Mail Merge
+
+With the nested mail merge functionality it is possible to add all items to a single line. This is achieved by adding the group and regular fields to a single paragraph.
```
Comment (vladislav-todorov) [inline libraries/radwordsprocessing/editing/mail-merge.md:187]:
```suggestion
With the nested mail merge functionality, it is possible to add all items to a single line. This is achieved by adding the group and regular fields to a single paragraph.
```

File: libraries/radwordsprocessing/editing/mail-merge.md [hunk 4]
```diff
@@ -93,6 +93,109 @@ __Example 4__ performs the mail merge operation over a previously defined templa
...
+
+>caption Figure 1: Mail Merging on single row and the results
```
Comment (vladislav-todorov) [inline libraries/radwordsprocessing/editing/mail-merge.md:189]:
```suggestion
>caption Figure 1: Mail Merging on a single row and the results
```

File: libraries/radwordsprocessing/editing/mail-merge.md [hunk 5]
```diff
@@ -93,6 +93,109 @@ __Example 4__ performs the mail merge operation over a previously defined templa
...
+
+![Rad Words Processing mail merge](images/RadWordsProcessing_MailMerge_01.png)
+
+If you want separate the items on several rows you need to clowse the group on the next row
```
Comment (vladislav-todorov) [inline libraries/radwordsprocessing/editing/mail-merge.md:193]:
```suggestion
If you want to separate the items into several rows you need to close the group on the next row
```

File: libraries/radwordsprocessing/editing/mail-merge.md [hunk 6]
```diff
@@ -93,6 +93,109 @@ __Example 4__ performs the mail merge operation over a previously defined templa
...
+
+The nested mail merge functionality is supported from R1 2022. It allows you to merge data sources that contain nested data. For example, your business object can contain a list of other objects and this functionality allows accessing the properties of the underlying objects. In order to use the underlying objects, you need to  declare a group. Currently, the following group tags are supported. 
+
+* BeginGroup/EndGroup 
+* TableStart/TableEnd
+* RangeStart/RangeEnd
+* GroupStart/GroupEnd
+
+
+Currently, all tags work equally and more than one option exist in order to improve the readability of the documents. The below example demonstrates how you can use the nested mail merge.
```
Comment (velikovmartin) [inline libraries/radwordsprocessing/editing/mail-merge.md:106]: existS

**Review by velikovmartin — CHANGES_REQUESTED** [review]

File: libraries/radwordsprocessing/editing/mail-merge.md [hunk 7]
```diff
@@ -93,6 +93,109 @@ __Example 4__ performs the mail merge operation over a previously defined templa
...
+
+{{endregion}}
+
+Now you need to add the fields using the specific supported names. In this example we are adding the fields to the table and we will use the TableStart/TableEnd tags, but this is not mandatory and you can use a tag of your choosing.
+
+#### __[C#] Example 6: Perform nested mail merge__
+
+{{region cs-radwordsprocessing-editing-mail-merge_5}}
+    var document = new RadFlowDocument();
+	var editor = new RadFlowDocumentEditor(document);
```
Comment (velikovmartin) [inline libraries/radwordsprocessing/editing/mail-merge.md:156]: There is additional tab here.

**Review by velikovmartin — CHANGES_REQUESTED** [review]

File: libraries/radwordsprocessing/editing/mail-merge.md (hunk 6 above)
Comment (DKaramfilov) [reply libraries/radwordsprocessing/editing/mail-merge.md:106]: fixed

File: libraries/radwordsprocessing/editing/mail-merge.md (hunk 7 above)
Comment (DKaramfilov) [reply libraries/radwordsprocessing/editing/mail-merge.md:156]: fixed

**Pull request merged**